src/games/strategy/triplea/ui/TripleAFrame.java

```diff
@@ -557,8 +557,6 @@ public void stopGame() {
     if (uiContext == null) {
       return;
     }
-    menu.dispose();
```

Comment:
It's not a call to a swing method. I thought it was.

So, removing this call means that the game notes window will not be closed when you close the main game window. But, we also have other windows that can be opened and are not included, so the game notes window is unique in having this behavior.

If we would like to restore this behavior, IMO we should cut an issue, I think we would largely be rewriting this, and referencing this diff is a good enough of a starting point. Meanwhile, the behavior of window closing is consistent (they stay open, unit help notes is an example)